core/iwasm/interpreter/wasm_loader.c

```diff
@@ -6,6 +6,8 @@
 #include "wasm_loader.h"
 #include "bh_common.h"
 #include "bh_log.h"
+#include "platform_api_extension.h"
+#include "platform_api_vmcore.h"
```

Comment:
Seems no need to include them, will the compiler report error if they are not included? 

If to include, include "bh_platform.h" instead. `platform_api_extension.h` and `platform_api_vmcore.h` are internal files of platform layer.